### PR TITLE
Changes to facilitate running in an IDE (outside of docker).

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,22 @@ tests are written in Kotlin and use the standard `kotlintest` testing framework.
...
+                                                                                                                                                                                              
+    make services                                                                                                                                                                             
+                                                                                                                                                                                              
+Then insert into your local hosts file the names, IP addresses of the kafka and                                                                                                               
+hbase containers:                                                                                                                                                                             
+                                                                                                                                                                                              
+    ./hosts.sh
```
Comment (dan-hill2802) [inline README.md]: I don't see this file anywhere

Comment (DanielChicot) [reply README.md]: Looks like I forgot to stage that one. It's there now. Thanks for spotting that.

Comment (dan-hill2802) [reply README.md]: maybe this should also be run through a `make` command, for consistency

Comment (DanielChicot) [reply README.md]: Done.

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**